### PR TITLE
Allow use of CoreTests suite for non-transactional databases

### PR DESCRIPTION
**Pull request by MMatten**
Split out the `CommitAndRollback` test from the `CoreTests` suite so that this suite can be used for non-transactional databases (such as Informix running in Informix, rather than ANSI, mode).

### CODE REVIEW

**Comment by javornikolov** [comment]
Is there any difference between `non-transactional` (informix) mode and `autocommit=true`?

File: FitNesseRoot/DbFit/AcceptanceTests/CoreTests/CommonSuite/CommitAndRollback/content.txt [hunk 1]
```diff
@@ -1,35 +0,0 @@
-!|Rollback|
```
Comment (javornikolov) [inline FitNesseRoot/DbFit/AcceptanceTests/CoreTests/CommonSuite/CommitAndRollback/content.txt]: As far as I see this file is removed, but not moved to somewhere else. Should it be that way?

File: FitNesseRoot/DbFit/AcceptanceTests/JavaTests/Db2Tests/FlowMode/properties.xml [hunk 2]
```diff
@@ -11,6 +11,7 @@
 <Suite/>
 <Suites>(Shared Tests)</Suites>
 <SymbolicLinks>
+<CommitAndRollback>.DbFit.AcceptanceTests.LibraryTests.CommitAndRollback</CommitAndRollback>
```
Comment (javornikolov) [inline FitNesseRoot/DbFit/AcceptanceTests/JavaTests/Db2Tests/FlowMode/properties.xml]: Is `LibraryTests` a good name?

**Comment by javornikolov** [comment]
Hmm, interesting: `LibraryTests` folder is missing, there are symbolic links to them in our tests. Yet, TravisCI check have passed successfully...

**Comment by MMatten** [comment]
The moved `CommitAndRollback` test now added again.

File: FitNesseRoot/DbFit/AcceptanceTests/JavaTests/Db2Tests/FlowMode/properties.xml (hunk 2 above)
Comment (MMatten) [inline FitNesseRoot/DbFit/AcceptanceTests/JavaTests/Db2Tests/FlowMode/properties.xml]: There's probably a better name out there somewhere. I was lacking inspiration at the time. Any offers accepted!

The idea was to create a library of tests that can be reused/included where appropriate (where the database supports the feature). Half way between the common `CoreTests` and the tests that have been duplicated, for different DB types, that we still have hanging around.

**Comment by MMatten** [comment]
> Is there any difference between non-transactional (informix) mode and autocommit=true?

I think the behaviour is the same.

**Comment by javornikolov** [comment]
> The moved `CommitAndRollback` test now added again.

Great! I still wonder why tests didn't fail providing that this piece was missing?

**Comment by MMatten** [comment]
I'll try running the tests from the FitNesse UI. May be a bug in FitNesse?

**Comment by javornikolov** [comment]
> May be a bug in FitNesse?

Maybe. As far as I see - links pointing to nowhere are not listed in the page content at all.

**Comment by MMatten** [comment]
When I run the tests (e.g. `DerbyTests.FlowMode`) via the web browser interface when the `CommitAndRollback` directory is missing then there are no errors displayed - the suite passes.

We could create an issue on the FitNesse project if we want to follow this up.

**Comment by javornikolov** [comment]
> We could create an issue on the FitNesse project if we want to follow this up.

Sounds like a plan.

**Pull request merged**